examples/deadline/All-In-AWS-Infrastructure-Basic/python/package/lib/compute_tier.py

```diff
@@ -117,6 +118,12 @@ def __init__(self, scope: Construct, stack_id: str, *, props: ComputeTierProps,
             user_data_provider=UserDataProvider(self, 'UserDataProvider')
         )
 
+        # This is an example of how you can set up a fleet to be enabled for use with the Session
```

Comment:
Maybe add a note that for this to work they either have to be using an AMI that is based on Amazon Linux 2 or an Amazon-provided Windows Server, or have installed the SSM-Agent and set it to run on startup?

Reply:
This seemed like a better fit for the class's docstring, so I added it there.

Reply:
I agree.